app/scripts/controllers/transactions/index.test.js

```diff
@@ -3426,7 +3426,7 @@ describe('Transaction Controller', function () {
       });
     });
 
-    it('ignores new transactions if hash matches existing transaction', async function () {
```

Comment:
Would this test be clearer if it added the `incoming` transaction type to the `existingTransaction` but we added an additional exiting transaction where the type isn't incoming and then we assert that only one of the new incoming transactions is added?

We could simplify the name to: `ignores new transactions if hash matches existing incoming transaction`?